Dockerfile.win10.min

```diff
@@ -87,7 +75,7 @@ RUN choco install git docker unzip -y
 #
 # Installing python
 #
-ARG PYTHON_VERSION=3.10.11
+ARG PYTHON_VERSION=3.8.10
```

Comment:
I forgot the reasoning for the downgrade of python, could you please explain again?

Reply:
It's throw compilation errors, by downgrading it we were able to bypass it.